core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -989,8 +1012,11 @@ protected Query doToQuery(QueryShardContext context) throws IOException {
         }
 
         queryParser.setDefaultOperator(defaultOperator.toQueryParserOperator());
+        queryParser.setType(type);
         if (tieBreaker != null) {
             queryParser.setGroupTieBreaker(tieBreaker);
+        } else {
+            queryParser.setGroupTieBreaker(type.tieBreaker());
```

Comment:
The `tie_breaker` overrides the value embedded in `type` so it is not needed. We always set the `tie_breaker` in the multi match query builder when it's needed.